src/kili/mutations/label/__init__.py

```diff
@@ -154,61 +155,49 @@ def append_to_labels(
         result = self.auth.client.execute(GQL_APPEND_TO_LABELS, variables)
         return format_result("data", result, Label)
 
-    @typechecked
-    def append_labels(
-        self,
-        json_response_array: List[dict],
-        author_id_array: Optional[List[str]] = None,
-        label_asset_external_id_array: Optional[List[str]] = None,
-        label_asset_id_array: Optional[List[str]] = None,
-        seconds_to_label_array: Optional[List[int]] = None,
-        label_type: str = "DEFAULT",
-        project_id: Optional[str] = None,
-    ):
-        """Append a label to an asset.
+    def append_labels(self, labels: List[AppendLabelData], label_type: LabelType = "DEFAULT"):
```

Comment:
should be typechecked imho (this is what we do for public methods)

Reply:
we should not force the user to use a TypeDict. So we should have
```suggestion
    def append_labels(self, labels: List[Dict], label_type: LabelType = "DEFAULT"):
```